src/autopas/AutoPas.h

```diff
@@ -452,7 +452,9 @@ class AutoPas {
   [[nodiscard]] AcquisitionFunctionOption getAcquisitionFunction() const { return _acquisitionFunctionOption; }
 
   /**
-   * Set acquisition function for tuning
+   * Set acquisition function for tuning.
+   * For possible acquisition function choices see options::AcquisitionFunctionOption::Value.
+   * @note This function is only relevant for the bayesian search.
```

Comment:
```suggestion
   * @note This function is only relevant for the bayesian based searches.
```
Da wird es voraussichtlich mehrere geben ;)